test/common/http/codec_impl_fuzz_test.cc

```diff
@@ -318,6 +331,28 @@ class HttpStream : public LinkedObject<HttpStream> {
       ENVOY_LOG_MISC(debug, "Request stream action on {} in state {} {}", stream_index_,
                      static_cast<int>(request_.stream_state_),
                      static_cast<int>(response_.stream_state_));
+      if (stream_action.has_dispatching_action()) {
+        // Simulate some response action while dispatching request headers, data, or trailers.
```

Comment:
Is the idea to model what happens when a filter decides to do something like `encodeData()` while in the middle of a `decodeHeaders()` callback in a single dispatch call stack? Or something else?

Would be useful to clarify this in the comments here and in the `.proto`.

Reply:
I did some research. It's effectively trying to recreate the local reply callstack. The testing added in the fix (https://github.com/envoyproxy/envoy/pull/9996) simulates like this, but I think in an ideal case we want to fuzz local replies. I think the bug was originally triggered by pending H/2 frames that were sent in that state. @mkbehr Do you have any stack traces you could point me towards? Thanks! 

I'm wondering if this would be better triggered in HCM.